### DISCUSSION TITLE
Connector to Azure Cognitive Search

### DISCUSSION THREAD

**Discussion by boble23** [FlowiseAI/Flowise#2325] [Q&A]
Hi everyone,

I recently started using Flowise and am impressed with its capabilities. However, I am unable to find the Azure Cognitive Search service within the category vector stores. Having the ability to load an existing index would be a great functionality.

I noticed that LangChain has an integration with [Azure AI Search](https://python.langchain.com/docs/integrations/vectorstores/azuresearch/), which made me wonder if there might be a specific reason why this option isn’t available in Flowise. Any insights or guidance on this matter would be greatly appreciated.

Thank you

**Comment by shamyne** [comment]
@boble23 did you get this resolved?
